using-babelfish/getting-started.md

```diff
@@ -16,15 +16,27 @@ After playing with the dashboard, you will probably want to get Babelfish runnin
 
 ### Running with Docker \(recommended\)
 
-The easiest way to run the _bblfshd_ daemon is using Docker. You can start it with the following command:
+The easiest way to run the _bblfshd_ daemon is using Docker. You can do it in
+stateless mode, meaning that all installed drivers will be wiped once you stop
```

Comment:
> drivers will be wiped once you stop the container
Since the commands don't involve `--rm` it would happen only on explicit removal of `bblfshd` container. Not sure that this makes any difference for the end user, but maybe worth mentioning.

Reply:
Good point, fixed.